packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -51,20 +51,36 @@ class FramesBarPlotly {
         // Hide ticks by using font color of bgColor.
         tickfont: Font(
           color: 'white',
+          size: 1,
         ),
         rangemode: 'nonnegative',
         autorange: true,
       ),
       yaxis: AxisLayout(
         title: 'Milliseconds',
-        fixedrange: true,
+        tickformat: '.0f',
+        type: 'log',
+        range: [0, 2],
+        nticks: 3,
+        tickmode: 'array',
+        tickvals: [
+          1,
+          10,
```

Comment:
can we tweak these so a line hits at 8 or 16 ms?

Reply:
Unable to display those ticks marks.  However, I did add code to display 2 highwater grids for 8 and 16 ms.

![image](https://user-images.githubusercontent.com/2055873/53888861-44ba6f00-3fda-11e9-93cc-33b1082838b5.png)

Only trick is how to display the ".... 8 ms" and "---- 16 ms".  Could display as unused traces with those colors and strokes so they show up in legend.  However, would have to increase the chart height.  Out of all of them like the last one?  Could add the highwater marks as checkboxes above the chart?

![image](https://user-images.githubusercontent.com/2055873/53900969-9cfd6b00-3ff2-11e9-9b0e-d40bbbd6eb49.png)

or

![image](https://user-images.githubusercontent.com/2055873/53901103-f5346d00-3ff2-11e9-9691-253ed4bc0e2d.png)

or

![image](https://user-images.githubusercontent.com/2055873/53901416-c8348a00-3ff3-11e9-8b25-3a419518f685.png)

Do we want those highwater legends above or below the bars (z-order).  Below the bars they kind of disappear above they look okay.  Was thinking if they're above the bars then control the highwater lines using checkboxes for 8 ms and 16 ms.  Will talk Tao. For now I'll add the grid lines.